runtime/src/main/java/org/corfudb/runtime/view/stream/AbstractQueuedStreamView.java

```diff
@@ -911,6 +913,8 @@ synchronized void seek(long globalAddress) {
      */
     @Data
     static class StreamCheckpoint {
+        public static StreamCheckpoint UNINITIALIZED = new StreamCheckpoint();
+        public static StreamCheckpoint INITIALIZED = new StreamCheckpoint();
```

Comment:
Could they be final? Looks we don't change their references.

Reply:
Yep.